src/main/resources/static/download.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Download file</title>
+</head>
+<body>
+    <a href="files/excel_example.xlsx">Download excel example</a>
```

Comment:
```suggestion
    <a href="files/excel_sample.xlsx">Download Excel sample</a>
```

to avoid potential intersection with JBehave Examples

Reply:
fixed